src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
```

Comment:
null is redundant since you return falsy value anyway when you haven't found user
```suggestion
  return usersFromServer.find(user => user.id === userId);
```